src/discord_utils.py

```diff
@@ -31,6 +31,26 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def filepath_builder(message_id: int, attachment: Attachment) -> str:
```

Comment:
Maybe better to rename this slightly more semantically to "attachment_local_filepath" or something to better reflect that it constructs a unique local filepath from an attachment. Also maybe add a comment/docstring explaining this